requirements/doc.in

```diff
@@ -4,5 +4,5 @@ doc8                      # reStructuredText style checker
 edx_sphinx_theme          # edX theme for Sphinx output
 readme_renderer==0.7.0    # Validates README.rst for usage on PyPI
 Sphinx                    # Documentation builder
-sphinxcontrib-napoleon    # Google Style docstring support for sphinx
+sphinxcontrib-napoleon==0.6.1     # Google Style docstring support for sphinx
```

Comment:
@nasthagiri: Had to pin this to get doc tests to pass in Travis.

Reply:
You shoudn't need this package anymore, it was added to Sphinx proper back in version 1.3; just need to update the name of the extension in `conf.py`: http://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html

Reply:
FYI: I created ENT-1293 for this clean-up.  I have many upgrades of edx-drf-extensions to get out that are blocked.